docs/tutorials/zoom-activity-dashboard.md

```diff
@@ -0,0 +1,277 @@
+---
+description: Using Airbyte and Tableau
+---
+
+# Visualizing the Time Spent by Your Team in Zoom Calls
+
+In this article, we will show you how you can understand how much your team leverages Zoom, or spends time in meetings, in a couple of minutes. We will be using [Airbyte](https://airbyte.io) \(an open-source data integration platform\) and [Tableau](https://www.tableau.com) \(a business intelligence and analytics software\) for this tutorial.
+
+Here is what we will cover:
+
+* Step 1: Setting up data replication from Zoom to a PostgreSQL database using the Airbyte Zoom connector
+* Step 2: Connecting the PostgreSQL database to Tableau
+* Step 3: Creating charts in Tableau with Zoom data
+
+We will produce the following charts in Tableau:
+
+* Evolution of the number of meetings per week in a team
+* Evolution of the number of hours a team spends in meetings per week
+* Listing of team members with the number of meetings per week and number of hours spent in meetings, ranked
+* Evolution of the number of webinars per week in a team
+* Evolution of the number of hours a team spends in webinars per week
+* Evolution of the number of participants for all webinars in a team per week
+* Listing of team members with the number of webinars per week and number of hours spent in meetings, ranked 
+
+Let’s get started by replicating Zoom data using Airbyte.
+
+## Step 1: Replicating Zoom data to PostgreSQL
+
+### Launching Airbyte
+
+In order to replicate Zoom data, we will need to use [Airbyte’s Zoom connector](https://docs.airbyte.io/integrations/sources/zoom). To do this, you need to start off Airbyte’s web app by opening up your terminal and navigating to Airbyte and running:
+
+`docker-compose up`
+
+You can find more details about this in the [Getting Started](https://docs.airbyte.io/getting-started) tutorial.
+
+This will start up Airbyte on `localhost:8000`; open that address in your browser to access the Airbyte dashboard.
+
+![](../.gitbook/assets/01_airbyte-dashboard.png)
+
+In the top right corner of the Airbyte dashboard, click on the **+ new source** button to add a new Airbyte source. In the screen to set up the new source, enter the source name \(we will use airbyte-zoom\) and select **Zoom** as source type.
```

Comment:
this assumes that the user is not seeing the onboarding screen. we should say something about skipping onboarding